foundations/javascript_basics/developer_tools_2.md

```diff
@@ -1,6 +1,11 @@
 ### Introduction
 Knowing how to use the Developer Tools inside your browser is an important skill for any web developer. The Developer Tools are useful for running JavaScript code, editing HTML and CSS styles without having to refresh the page, and viewing performance data. They also save you lots of time. Getting started couldn't be any easier. You should already be familiar with using them for debugging and working on HTML and CSS, so let's take a look at how they can benefit us when writing JavaScript.
 
+### Learning Outcomes
+By the end of this lesson, you should be able to:
+
+* Copy JavaScript Path in Chrome DevTools
```

Comment:
I'm not sure if one learning outcome is a good idea to add. We should probably consider adding a handful of these. Can you add a few of them to cover more of the lesson? 

Copying javascript paths is pretty low on the important-to-learn list.

Reply:
Yes, I can come up with a few more ideas for this Learning Outcome section. It will take some time.

Reply:
How are we doing on this?

Reply:
I added a few more points in the Learning Outcome section. It looks a lot more full now.